src/support/timefuncs.c

```diff
@@ -29,30 +29,29 @@ extern "C" {
 #endif
 
 #if defined(_OS_WINDOWS_)
```

Comment:
Thanks. I figured I'd ask in case there was a particular style the Julia source was moving towards.